### ISSUE TITLE
Releases and docs are difficult to understand

### ISSUE THREAD

**Issue by ysalmon** [sdaps/sdaps#145]
Greetings,

troubleshooting my problems in #144, I came to the conclusion that the current state of releases of SDAPS on the website is quite cumbersome :

- the stable PPA has version 1.9.3 (new sdaps class) in Bionic and 1.2.x in earlier channels (old sdaps class) ;
- however the [stable branch](https://github.com/sdaps/sdaps/tree/stable) in the git repo seems to be the 1.2 line (old sdaps class)
- on the website, the "LaTeX support" link in the menu leads to [the documentation of the newer class](https://sdaps.org/class-doc/), but the Features section in the front page has a link to [an example with the old class](https://sdaps.org/documentation/latex/)
- there does not seems to be an indication of which feature was added/changed with which version of sdaps

Surveys created for the old sdaps class are likely to stay around for some time ; it would be nice if it were less complicated to understand that such a survey needs the older version 1.2 of sdaps and to get that version.

**Comment by benzea** [comment]
Yeah, this is a messup. The issue at hand is that 1.2 is so outdated that it is effectively unusable, but the current "unstable" release is not quite there yet.

Basically, the idea was that we need to get 1.9.3 into shape for 2.0 ASAP and we would solve the issues by doing so.
